lib/src/main/kotlin/com/swmansion/starknet/signer/Signer.kt

```diff
@@ -23,4 +23,9 @@ interface Signer {
      * Public key used by a signer.
      */
     val publicKey: Felt
+
+    /**
+     * Private key used by a signer
+     */
+    val privateKey: Felt
```

Comment:
Why do we need this?

Reply:
Mostly for testing purposes.

Reply:
Private key can't be in interface. It might not exist for other signers.